data_pipeline/generic_web_api/generic_web_api_config.py

```diff
@@ -145,105 +183,11 @@ def __init__(
         ) if auth_type and auth_conf_list else None
 
 
-class DynamicURLManager:
-    def __init__(
-            self,
-            url_excluding_configurable_parameters: str,
-            from_date_param: str = None,
-            to_date_param: str = None,
-            date_format: str = None,
-            next_page_cursor: str = None,
-            page_number_param: str = None,
-            page_size_param: str = None,
-            page_size: int = None,
-    ):
-        self.url_excluding_configurable_parameters = (
-            url_excluding_configurable_parameters
-        )
-        self.from_date_param = from_date_param
-        self.to_date_param = to_date_param
-        self.date_format = date_format
-        self.next_page_cursor = next_page_cursor
-        self.page_number_param = page_number_param
-        self.page_size_param = page_size_param
-        self.page_size = page_size
-
-    def get_url(
-            self,
-            from_date: datetime = None,
-            to_date: datetime = None,
-            page_number: int = None,
-            cursor: str = None,
-            page_size: int = None
-    ):
-        start_date = datetime_to_string(from_date, self.date_format)
-        end_date = datetime_to_string(to_date, self.date_format)
-        param_dict = dict((key, value) for key, value in [
-            (self.from_date_param, start_date),
-            (self.next_page_cursor, cursor),
-            (self.to_date_param, end_date),
-            (self.page_number_param, page_number),
-            (self.page_size_param, page_size or self.page_size)
-            ] if key and value)
-        url = self.url_excluding_configurable_parameters
-        if "?" in url:
-            if url.strip().endswith("&") or url.strip().endswith("?"):
-                url_separator = ""
-            else:
-                url_separator = "&"
-        else:
-            url_separator = "?"
-
-        params = parse.urlencode(
-            {
-                key: value
-                for key, value in param_dict.items() if key and value
-            }
+class ResponsePathKey:
+    def __init__(self, path_level: str):
+        self.key = path_level if isinstance(path_level, str) else None
+        self.is_variable = (
+            True if isinstance(path_level, dict) and
+            path_level.get('isVariable')
```

Comment:
The sample config doesn't seem to include any not-commented out config that is making use of `isVariable`. Maybe we can remove it?

(If we are starting to use variables - maybe it's time to give it the configuration syntax a name, e.g. MCL - Michael Configuration Language :-) )